python/EventStore.py

```diff
@@ -1,3 +1,6 @@
+
+from past.builtins import xrange
```

Comment:
I would rather not import the past to the future but the future to the past. So not give python 3 features from python 2, but rather assume it's python 2 and give it python 3 features. The code should read as Python 3 (so no visible xrange) so it's "easier" to drop python 2.

Reply:
Good point. Done.